Dockerfile

```diff
@@ -8,4 +8,17 @@ RUN \
   rm -rf /var/lib/apt/lists/* && \
   rm -rf /var/cache/oracle-jdk8-installer
 
+RUN \
+  apt-get update && \
+  apt-get install -y libnss3 libfontconfig
+
+RUN \
+  apt-get update && \
+  apt-get install -y libxss1 libappindicator1 libindicator7 libasound2 fonts-liberation xdg-utils && \
+  wget -N https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && \
+  sudo dpkg -i google-chrome-stable_current_amd64.deb && \
+  sudo apt-get -f install -y && \
+  sudo dpkg -i google-chrome-stable_current_amd64.deb
```

Comment:
quote from some other resources:

```
That is because .deb files are installed with a different command!

In order to install a package from a .deb file you should use the following command:  sudo dpkg -i google-chrome-stable_current_amd64.deb in your case.

As I remember it, that's all installing chrome requires, that said, as a general fyi sometimes dpkg will not work the first time and it will inform you to run sudo apt-get install -f. But after you do just that, you will be able to run dpkg -i again and it will work.
```